eclair-core/src/main/scala/fr/acinq/eclair/payment/PaymentLifecycle.scala

```diff
@@ -218,6 +214,18 @@ object PaymentLifecycle {
   case class UnreadableRemoteFailure(route: Seq[Hop]) extends PaymentFailure
   case class PaymentFailed(id: UUID, paymentHash: ByteVector32, failures: Seq[PaymentFailure]) extends PaymentResult
 
+  /**
+   * Options to help build the final payload of the payment route.
+   */
+  sealed trait PaymentOptions {
```

Comment:
Can we find a name that better conveys the fact that this targets the final node?